### ISSUE TITLE
Ban in telegram

### ISSUE THREAD

**Issue by ykslol** [vysheng/tg#1560]
Hi.
Wonderful cliente, but if i register account via this CLI, telegram instantly ban this accounts.
If register account via oficial clients and then login via this CLI - all is OK.

May be someone know how to resolve this issue. Thanks

**Comment by Paulchen232** [comment]
You created a new account at the make command with a number and specified the pin from the SMS or the phone call. And then?

**Comment by Paulchen232** [comment]
What do you see after you started telegram-cli? What is the start command?

**Comment by ykslol** [comment]
1.phone number:
2. register (Y/n):
3.first name:
4.last name:
5. code ('CALL') for phone code):

**Comment by Paulchen232** [comment]
I do not understand. What is your startup code? With me, e.g ~/tg/bin/telegram-cli -C -W -s ~/tg/test.lua -k ~/tg/tg-server.pub -L ~/tg/logfile.log

**Comment by ykslol** [comment]
telegram-cli -W -k server.pub - p profile_2

**Comment by Paulchen232** [comment]
In which folder is server.pub? You have this not specified.

**Comment by Paulchen232** [comment]
And there is a space between - and p. Is that just a copy error?

**Comment by Paulchen232** [comment]
Whats your problem? I have think that all going now.

**Comment by ykslol** [comment]
If i try to registrate account via this client - telegram banned this account. But if i registrate via other client and then login via this cli - all is ok. how can i registrate telegram account via this client without ban? Or may be u know how to registrate account in telegram via linux shell.

**Comment by Paulchen232** [comment]
I think I knows to less. How you make the registraten with telegram-cli? I only ?login? with `make`with an old registration.

**Comment by ykslol** [comment]
I started telegram-CLi. Then it ask me for the phone number. I put new phone number for registration. Then it ask me dont i  want to registrate new account, i answer yes. Then i pun my first and last name and then i put the code which my phone resieved. After i put the code i logged in and in 1-10 sec telegram ban this account.

**Comment by Paulchen232** [comment]
Ok I try this too.

**Comment by Paulchen232** [comment]
It works, sorry. I use my Handy number, add "0" and let my call for the Code. My Startcode was "~/tg/bin/telegram-cli -W -s ~/tg/test.lua -k ~/tg/tg-server.pub -p profil1". My Version is 1.4.1. Perhaps you send from telegram-cli ban a picture link or the picture to Telegram username @GoOssi.

**Comment by ykslol** [comment]
So i started telegram-Cli, put number, get sms with code and after registration account become deactivated == banned
```
telegram-cli -W -k /opt/tg/server.pub -p profile_1 -c config -v 4
change_user_group: can't find the user telegramd to switch to
Telegram-cli version 1.4.1, Copyright (C) 2013-2015 Vitaly Valtman
Telegram-cli comes with ABSOLUTELY NO WARRANTY; for details type `show_license'.
This is free software, and you are welcome to redistribute it
under certain conditions; type `show_license' for details.
Telegram-cli uses libtgl version 2.1.0
Telegram-cli includes software developed by the OpenSSL Project
for use in the OpenSSL Toolkit. (http://www.openssl.org/)
 *** 1541531247.280667 adjusting CLOCK_MONOTONIC delta to -0.587536
 *** 1541531247.332554 adjusting CLOCK_MONOTONIC delta to -0.590192
 *** 1541531247.807820 adjusting CLOCK_MONOTONIC delta to 0.543261
 *** 1541531247.826769 adjusting CLOCK_MONOTONIC delta to 0.625269
 *** 1541531248.050743 adjusting CLOCK_MONOTONIC delta to 0.665799
phone number: 8617123624401
register (Y/n): y
first name: John
last name: Johnson
code ('CALL' for phone code): 66499
> 
> 
 *** 1541531335.259072 error for query 'get state' #6620826613667390464: #401 USER_DEACTIVATED
 *** 1541531335.259110 login problem: error #71 (RPC_CALL_FAIL 401: USER_DEACTIVATED)
 *** 1541531335.259128 login failed
 *** 1541531335.259144 login error #71: RPC_CALL_FAIL 401: USER_DEACTIVATED
 *** 1541531335.259160 you can relogin by deleting auth file or running telegram-cli with '-q' flag
```

**Comment by Paulchen232** [comment]
Are you really root? [#81](https://github.com/luckydonald/pytg/issues/81)

**Comment by ykslol** [comment]
yes i run cli from root. But if i registrate account via official client and then login via this cli also run from root - login is ok.

**Comment by mmbug** [comment]
did you solve this? i have the same problem

**Comment by Paulchen232** [comment]
https://telegram.org/faq_spam

**Comment by ykslol** [comment]
mmbug, no =(
But i found a solution to run multiple instances of TG on PC. Via official client.
For on vps works 8 instances of TG (centos with GUI) registrated on virtual numbers. Not convenient as console cli, but better than nothing.
